packages/components/src/popover/index.tsx

```diff
@@ -416,10 +416,12 @@ const UnforwardedPopover = (
 	// document scrolls. Also update the frame offset if the view resizes.
 	useLayoutEffect( () => {
 		if (
-			// reference and floating are in the same document
+			// Reference and root documents are the same.
 			referenceOwnerDocument === document ||
-			// the reference's document has a view (i.e. window)
-			// and a frame element (ie. it's an iframe)
+			// Reference and floating are in the same document.
+			referenceOwnerDocument === refs?.floating?.current?.ownerDocument ||
```

Comment:
Do we need to add this to the deps or having `update` there is enough?

Reply:
Thanks for reviewing.

Because it's a ref I don't think it needs to be, and I don't think it'll be a value that can change 🤔 

I also didn't notice any issues, so I think lets leave it as it is for now.